.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+name: Unit Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      DIRECTORY_USERNAME: ${{ secrets.DIRECTORY_USERNAME }}
+      DIRECTORY_WEB_SERVICE: ${{ secrets.DIRECTORY_WEB_SERVICE }}
+      DIRECTORY_PASSWORD: ${{ secrets.DIRECTORY_PASSWORD }}
+      MARKLOGIC_LDR_USER: ${{ secrets.MARKLOGIC_LDR_USER }}
+      MARKLOGIC_LDR_PASSWORD: ${{ secrets.MARKLOGIC_LDR_PASSWORD }}
+      MARKLOGIC_LDR_URL: ${{ secrets.MARKLOGIC_LDR_URL }}
+      SPARQL_ROOT: ${{ secrets.SPARQL_ROOT }}
+      OWNCLOUD_USERNAME: ${{ secrets.OWNCLOUD_USERNAME }}
+      OWNCLOUD_PASSWORD: ${{ secrets.OWNCLOUD_PASSWORD }}
+      OWNCLOUD_WEB_SERVICE: ${{ secrets.OWNCLOUD_WEB_SERVICE }}
+      GITHUB: ${{ secrets.GITHUB }}
+
+    steps:
+
+    - name: Configure sysctl limits
+      run: |
+        sudo swapoff -a
+        sudo sysctl -w vm.swappiness=1
+        sudo sysctl -w fs.file-max=262144
+        sudo sysctl -w vm.max_map_count=262144
```

Comment:
Is the purpose of this to set an upper bound on the filesystem the cloud thingy spins up every time it runs CI?

Reply:
Those settings are needed for Elasticsearch to run. I guess it fails otherwise but yeah, that happens every time it spins up.